voice/bxml/verbs/speakSentence.md

```diff
@@ -68,7 +69,7 @@ Attributes:
  * `level`: (optional) defines the strength of emphasis to be applied, accepted values are: `strong`, `moderate` or `reduced`
 
 #### `<lang>`
-Specifies the natural language of the content.
+Specifies the natural language of the content. If you use a voice with an `en_US` locale and an `es-MX` lang, it will sound like an English-speaking American attempting to speak Spanish.
```

Comment:
It sounds kind of informal. What if it was something like:
If you use a voice with an `en_US` locale and an `es-MX` lang, it will speak Spanish with an English-speaker accent.
But your english is certainly better than mine so feel free to ignore this. :)

Reply:
haha, that's basically what I had before but Nick warned against using "accent", which I kinda agreed with.